speaker/layer2_controller.go

```diff
@@ -133,9 +134,24 @@ func (c *layer2Controller) ShouldAnnounce(l log.Logger, name string, toAnnounce
 	return "notOwner"
 }
 
+func getInterfaces(localNode string, l2Advertisements []*config.L2Advertisement) sets.String {
```

Comment:
Same, put below where it's used. See https://kentcdodds.com/blog/newspaper-code-structure for reference

Reply:
done